Resources/views/chart/visualizationChart.html.twig

```diff
@@ -1,7 +1,32 @@
-var data{{ id }} = new google.visualization.DataTable({{ data | json_encode | replace({"\"[new Date]": 'new Date', "[new Date]\"": ""}) | raw }});
+{% if chartType == 'Calendar' or chartType == 'Timeline' %}
+    {% set data = data | json_encode | replace({"\"new Date[[[": 'new Date(', "new Date\"": ""}) | replace({"]]]\"": ')', "]]]\"": ")"}) | replace({"id\":\"role_tooltip": "role\": \"tooltip", "id\":\"role_tooltip": "role\": \"tooltip"}) %}
```

Comment:
Yes I'm pretty sure there is a better way but I search for quickest way that do not interact with tooltip or string content (I had issue by generating new Date(2016, 2, 1) and strings ending by ')'.

I plan to enhance this point directly in the Data php classes when I had enough time to make a snippet sufficiency strong to convert directly Datetime php objects in the good Json return and avoid using json_encode and replace in twig.